node/shard/src/types.rs

```diff
@@ -121,3 +134,31 @@ impl SignedBlock for SignedShardBlock {
         1
     }
 }
+
+#[derive(Hash, Debug, Serialize, Deserialize, PartialEq, Eq, Clone)]
```

Comment:
also why Hash here? we use our own hashing

Reply:
Fixed in 00f85e7